### DISCUSSION TITLE
Recruiting for GitHub's Accessibility Discussion Panel!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#128172] [Accessibility]
Are you a developer with a disability? Do you use assistive technology? Then [GitHub](https://www.linkedin.com/company/github/) wants to hear from you!

By joining our Inclusive Research Panel, your perspective and experiences will help shape and influence the accessibility of the platform as we work on new products and features. The panel consists of customers with disabilities who've expressed interest in participating in compensated studies to provide feedback and insights on the user experience of existing and new products.

If this interests you, please complete [this brief survey](https://forms.gle/23Jgw5tRwPLZHNTs5) so that we may follow up with you!

**Comment by abdessalems** [comment]
REPONSE 1

**Comment by abdessalems** [comment]
REPONSE 2

**Comment by Zeus-Aphrodite** [comment]
Hello Thank you for your discussions.
I am blockchain and fullstack engineer with 6+ years of years of experience.
Thanks.

**Reply by ghost** [reply]
Hi, I am a full stack engineer with 5 years of experience.Thanks

**Reply by abrahem79** [reply]
> شكرا لك على هذه الفرصة. في هذا الوقت، لا أستخدم التكنولوجيا المساعدة، لكنني أقدر التزام GitHub بإمكانية الوصول.

**Reply by gamal2421** [reply]
yoo man can you help me at blockchain ?

**Reply by mdsajib1473** [reply]
May I ask how much time you needed to learn blockchain?

**Reply by ayesha030348-a11y** [reply]
This is a great initiative by GitHub. Inclusive research and direct feedback from developers who use assistive technology is essential for building truly accessible platforms. It’s encouraging to see accessibility being treated as a priority in product development. More insights and resources on accessible digital experiences can also be explored here: https://thelightroommodapk.com/

**Comment by abrahem79** [comment]
@abrahem79

**Comment by abrahem79** [comment]
#
``` @queenofcorgis

**Comment by ShamrockLee** [comment]
There are some blank options, e.g. options that literally read "Option 3" and "Option 8", in the survey. Such blank options might confuse people who use screen-reader without a visual display.

**Reply by Mohammedsapry** [reply]
Ok

**Comment by shehroz9712** [comment]
Yes, I’m a developer with a disability and I do use assistive technology in my daily workflow.
I’m very interested in joining the Inclusive Research Panel. I believe that inclusive design and accessibility improvements can make a real difference, and I’d love to contribute my experiences to help shape GitHub’s tools and features. I appreciate the opportunity to be part of something that could benefit not just me, but many others in the developer community.

Please share the survey link so I can complete it and move forward with the process.

**Reply by queenofcorgis** [reply]
@shehroz9712 the link is in the discussion above, but I'm [linking the survey here](https://forms.gle/23Jgw5tRwPLZHNTs5) too.

**Comment by Gefrei666** [comment]
<img width="256" height="256" alt="AppIcon" src="https://github.com/user-attachments/assets/87bd9b34-c52b-4c7d-b926-8c42d84d3f62" />

**Comment by cary-rowen** [comment]
"Alt+ArrowUp"
This description is really too redundant for screen reader users

**Comment by fsystemweb** [comment]
Hello,
I’m interested in joining GitHub’s Accessibility Discussion Panel. I have experience with frontend development and working with AA and AAA accessibility standards. I’d love to participate and contribute.

**Comment by lockfilement** [comment]
What a great opportunity to contribute to accessibility on GitHub! If you're a developer with a disability and use assistive technology, GitHub is interested in your perspective to improve the platform. By joining the Inclusive Research Panel, your experiences will help shape new products and features. The panel consists of customers with disabilities who participate in paid research to provide feedback on the user experience of existing and new products.

**Comment by jsoref** [comment]
- there's still an "Option 5" similar to the items mentioned in: https://github.com/orgs/community/discussions/128172#discussioncomment-12728302

I'm sighted, but I'm still confused.
